spec/models/stripe_wrapper_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'spec_helper'
+
+describe StripeWrapper do
+  describe StripeWrapper::Charge do
+    describe ".create" do
+      it "makes a successful charge", :vcr do
```

Comment:
Since StripeWrapper will use :vcr for all each examples, you could push it to a level above to `describe StripeWrapper::Charge, :vcr`. And delete from examples bellow.